src/promptflow/promptflow/executor/_service/contracts/tool_request.py

```diff
@@ -0,0 +1,24 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from pathlib import Path
+from typing import Any, Mapping
+
+from pydantic import BaseModel
```

Comment:
Cool! We don't need to parse json manually~

Reply:
Yes, it's convenient.